docs/static/releasenotes.asciidoc

```diff
@@ -22,6 +23,81 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-5-1]]
+=== Logstash 7.5.1 Release Notes
+
+* Mention the path of DLQ to indicate DLQ if full for which pipeline https://github.com/elastic/logstash/pull/11280[#11280]
+* Changed GemInstaller to don't blank  gemspec attribute https://github.com/elastic/logstash/pull/11340[#11340]
+* Support remove_field on metadata and tests [7.5 clean backport of #11334] https://github.com/elastic/logstash/pull/11342[#11342]
+* Fix: do not leak ThreadContext into the system https://github.com/elastic/logstash/pull/11356[#11356]
+* test codec against class name string to prevent class equivalence bug with a Delegator https://github.com/elastic/logstash/pull/11401[#11401]
+* update grok filter to 4.2.0 (for LS 7.5.1) https://github.com/elastic/logstash/pull/11432[#11432]
```

Comment:
```suggestion
* update grok filter to 4.2.0, adding `timeout_scope` option which can be used to reduce performance overhead https://github.com/elastic/logstash/pull/11432[#11432]
```

Reply:
do we need to call this out here if we are already calling it out in the plugin changelog below?